cartography/intel/okta/groups.py

```diff
@@ -82,11 +82,13 @@ def _get_okta_group_members(api_client, group_id):
 
 def transform_okta_group_list(okta_group_list):
     groups = []
+    groups_id = []
 
     for current in okta_group_list:
         groups.append(transform_okta_group(current))
+        groups_id.append(current.id)
```

Comment:
Correct me if I'm wrong, but it seems that the group ID already exists here: https://github.com/lyft/cartography/blob/6954bc838dc5c2a3d20e0d0c4fcb02d21395b6aa/cartography/intel/okta/groups.py#L102

Is this additional `groups_id` list still necessary?